packages/pn-personafisica-webapp/src/pages/__test__/Contacts.page.test.tsx

```diff
@@ -60,18 +54,15 @@ describe('Contacts page', () => {
     expect(mockActionFn).toBeCalledWith('mocked-recipientId');
   });
 
-  test('subtitle link properly redirects to profile page', () => {
```

Comment:
é stato cancellato il test subtitle link properly redirects to profile page. come mai?